qq

```diff
@@ -0,0 +1,1399 @@
+[33m1ea1a507[m[33m ([m[1;36mHEAD -> [m[1;32mrefactor/europeana-provider-base-class[m[33m)[m Add back default cursor from previous implementation
```

Comment:
Should this file be here?

Reply:
Definitely not! I think I was trying to use vim binding to exit something that wasn't responding to `q` :cry: Somehow I am notorious for committing useless files like this, thanks for catching it. It is removed in https://github.com/WordPress/openverse-catalog/pull/821/commits/f02a4741814ede69b5349b0988241a88263364f1